devito/ir/iet/analysis.py

```diff
@@ -80,8 +80,9 @@ def mark_parallel(analysis):
             # (d_1, ..., d_i) = 0
             is_sequential = False
             for dep in analysis.scopes[i].d_all:
-                if not ((dims[:-1] and any(dep.is_carried(d) for d in dims[:-1])) or
-                        all(dep.is_independent(d) for d in dims)):
+                test0 = dims[:-1] and any(dep.is_carried(d) for d in dims[:-1])
```

Comment:
this is aesthetics, right?

Reply:
Oh, yeah.  The brackets were breaking my brain for a while.  I didn't mean to actually commit it, but then did accidentally and thought it might not be a bad idea after all.

Happy to drop or put in a separate PR or whatever.

Reply:
it's a very good idea, it was horrible before. Let's keep it !